export.go

```diff
@@ -724,7 +737,7 @@ func getLilyAPI(ctx context.Context, apiAddr string, apiToken string) (lily.Lily
 	for {
 		select {
 		case <-tick.C:
-			api, closer, err := commands.GetAPI(ctx, apiAddr, apiToken)
```

Comment:
oooff sorry, didn't think about the effect here when I made this change.

Reply:
It's fine. I duplicated the code to avoid pulling Lotus in explicitly, but it's better to be independent as much as we can be